lib/contexts/security-v2.js

```diff
@@ -7,10 +7,17 @@ module.exports = {
     "EquihashProof2018": "sec:EquihashProof2018",
     "RsaSignature2018": "sec:RsaSignature2018",
     "RsaVerificationKey2018": "sec:RsaVerificationKey2018",
-
+    "capability": {"@id": "sec:capability", "@type": "@id"},
```

Comment:
Seems like we're missing `allowedAction` and `capabilityAction` terms like are in the new DID context:

https://github.com/w3c-ccg/did-spec/blob/gh-pages/did-v0.11.jsonld#L19